### PR TITLE
HDRP UI changes for XR + Minimal functional changeset

### PR DESCRIPTION
**Pull request by nerites**
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
My hdrp_xr branch is bogged down with the question of how compute shaders ought to be programmed for stereo. This branch contains only the minimal changeset for XR. Notably, I left in the VR override, such that HDRP still officially does not support VR. 
---
### Release Notes
+ UI integration of HDRP with XRGraphicsConfig
+ Used DrawFullScreen for final blit when stereo is enabled
+ Removed gameViewRenderMode and mirrorToGameView from XRGraphicsConfig
+ Execute current command buffers before start/stop multieye calls

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
Tests are currently running: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=xr_uichanges&unity_branch=trunk&automation-tools_branch=add-platform-filter

**Manual Tests**: What did you do?
Loaded modified SRP in a test HRDP scene.

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low.

**Halo Effect**: None, Low, Medium, High?
Low.

---
### Comments to reviewers
Notes for the reviewers you have assigned.
These changes are a subset of the changes in the hdrp_xr branch.

### CODE REVIEW

File: com.unity.render-pipelines.core/CoreRP/Common/XRGraphicsConfig.cs [hunk 1]
```diff
@@ -19,24 +19,22 @@ public class XRGraphicsConfig
...
-        public bool showDeviceView;
-        public GameViewRenderMode gameViewRenderMode;
-        
+
         public void SetConfig()
         { // If XR is enabled, sets XRSettings from our saved config
             if (!enabled)
-                Assert.IsFalse(enabled);
+                return;
+            Assert.IsTrue(enabled);
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.core/CoreRP/Common/XRGraphicsConfig.cs]: Shouldn't the assert be before the test?  This way user would be warned before the method silently returns.

**Comment by sebastienlagarde** [comment]
I am good with the PR, except the thing with the Assert as they are useless. See Julien comment. You will also need to solve conflict from master

File: com.unity.render-pipelines.core/CoreRP/Common/XRGraphicsConfig.cs (hunk 1 above)
Comment (nerites) [reply com.unity.render-pipelines.core/CoreRP/Common/XRGraphicsConfig.cs]: I will remove the asserts. If I place them before the return, then when VR is disabled, the assertions will fire, causing Katana tests to fail even if stereo is not intended to be enabled for those tests.

**Comment by nerites** [comment]
Removed assertions and extraneous commits. Kicked off katana again just to check for typos.

**Pull request merged**